Library/Homebrew/cmd/sh.rb

```diff
@@ -13,7 +13,7 @@ def sh
     ENV.activate_extensions!
 
     if superenv?
-      ENV.x11 = MacOS::X11.installed?
+      ENV.x11_if_installed
```

Comment:
Can you explain what this was/is supposed to do? I don't really understand this, but I'm not sure the replacement is equivalent.